client/web-sveltekit/src/routes/[...repo=reporev]/RepoMenu.svelte

```diff
@@ -0,0 +1,123 @@
+<script lang="ts">
+    import { openFuzzyFinder } from '$lib/fuzzyfinder/FuzzyFinderContainer.svelte'
+    import { reposHotkey } from '$lib/fuzzyfinder/keys'
+    import Icon from '$lib/Icon.svelte'
+    import KeyboardShortcut from '$lib/KeyboardShortcut.svelte'
+    import { getHumanNameForCodeHost } from '$lib/repo/shared/codehost'
+    import CodeHostIcon from '$lib/search/CodeHostIcon.svelte'
+    import { getButtonClassName } from '$lib/wildcard/Button'
+    import DropdownMenu from '$lib/wildcard/menu/DropdownMenu.svelte'
+    import MenuButton from '$lib/wildcard/menu/MenuButton.svelte'
+    import MenuLink from '$lib/wildcard/menu/MenuLink.svelte'
+    import MenuSeparator from '$lib/wildcard/menu/MenuSeparator.svelte'
+
+    export let repoName: string
+    export let displayRepoName: string
+    export let repoURL: string
+
+    export let externalURL: string | undefined
+    export let externalServiceKind: string | undefined
+</script>
+
+<DropdownMenu triggerButtonClass={getButtonClassName({ variant: 'text' })}>
+    <svelte:fragment slot="trigger">
+        <div class="trigger">
+            <CodeHostIcon repository={repoName} codeHost={externalServiceKind} />
+            <h2>
+                {#each displayRepoName.split('/') as segment, i}
+                    {#if i > 0}<span class="slash">/</span>{/if}{segment}
+                {/each}
+            </h2>
+        </div>
+    </svelte:fragment>
+
+    <MenuLink href={repoURL}>
+        <div class="menu-item">
+            <Icon icon={ILucideHome} inline />
+            <span>Go to repository root</span>
+            <KeyboardShortcut shortcut={{ key: 'ctrl+backspace', mac: 'cmd+backspace' }} />
+        </div>
+    </MenuLink>
+    <MenuButton on:click={() => openFuzzyFinder('repos')}>
+        <div class="menu-item">
+            <Icon icon={ILucideRepeat} inline />
+            <span>Switch repo</span>
+            <KeyboardShortcut shortcut={reposHotkey} />
+        </div>
+    </MenuButton>
+    <MenuLink href={repoURL + '/-/settings'}>
```

Comment:
I think you can do

```suggestion
    <MenuLink href="{repoURL}/-/settings">
```

No strong preference put it feels a little cleaner to me.

Reply:
I always forget about that syntax. Thanks!